diskwriter_windows.go

```diff
@@ -1,8 +1,15 @@
+//go:build windows
 // +build windows
 
 package fsutil
 
 import (
+	"fmt"
+	ioFS "io/fs"
```

Comment:
what's up with this package alias. `fs` doesn't even seem to be used elsewhere in the file.

Reply:
remnant from splitting up the previous PR. Changing it now.

Reply:
The only way we can import ```io/fs``` is if we rename that struct. I think the aliased import is probably the path with the least amount of code changes :smile: . What do you think?

Reply:
Ok, but make it all lowercase.

Reply:
Import was renamed. Please take another look.